build_system/clerk_driver.ml

```diff
@@ -131,8 +131,8 @@ let backend_extensions =
     Clerk_rules.C, ["c"; "h"; "o"];
     Clerk_rules.OCaml, ["ml"; "mli"; "cmi"; "cmo"; "cmx"; "o"; "cmxs"];
     Clerk_rules.Python, ["py"];
-    Clerk_rules.Java, ["java"; "class"];
-    Clerk_rules.Tests, ["catala_en"; "catala_fr"; "catala_pl"];
+    Clerk_rules.Java, ["java"; "class"; "jar"];
```

Comment:
This change seems unexpected, probably slipped in during a rebase ?

Reply:
Yes, fixing it.